pkg/spiceerrors/assert_on.go

```diff
@@ -0,0 +1,12 @@
+//go:build ci
+// +build ci
+
+package spiceerrors
+
+import "fmt"
+
```

Comment:
Add a comment that this is only enabled in CI builds (and maybe point to the other impl for prod builds)

Reply:
Added